src/runtimes.ts

```diff
@@ -20,8 +20,8 @@ export const isWorkerd =
 export const isDeno = !!globalThis.Deno;
 // https://nodejs.org/api/process.html#processrelease
 export const isLagon = !!globalThis.__lagon__;
-export const isNode = globalThis.process?.release?.name === "node";
 export const isBun = !!globalThis.Bun || !!globalThis.process?.versions?.bun;
+export const isNode = !isBun && globalThis.process?.release?.name === "node";
```

Comment:
Please see https://github.com/unjs/std-env/issues/100#issuecomment-1835834888. Bun runtime deliberately tries to mimic Node and utilities are depending on std-env `isNode` to use Node.js API which is available in this situation. We might need to do this part of change later perhaps with better named exports and more disucssion.